cornac/datasets/README.md

```diff
@@ -0,0 +1,154 @@
+    <td align="center">BIN<br>{0,1}</td>
+    <td align="center">&#10004;</td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><a href="https://cornac.readthedocs.io/en/latest/datasets.html#module-cornac.datasets.epinions">Epinions</a><br><a href="http://www.trustlet.org/downloaded_epinions.html"> source </a></td>
+    <td align="right">40,163</td>
+    <td align="right">139,738</td>
+    <td align="right">664,824</td>
+    <td align="center">INT<br>[1,5]</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td align="center">&#10004;</td>
+  </tr>
+  <tr>
+    <td><a href="https://cornac.readthedocs.io/en/latest/datasets.html#module-cornac.datasets.movielens">MovieLens 100k</a><br><a href="https://grouplens.org/datasets/movielens/"> source </a></td>
+    <td align="right">943</td>
+    <td align="right">1,682</td>
+    <td align="right">100,000</td>
+    <td align="center">INT<br>[1,5]</td>
+    <td align="center">&#10004;</td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><a href="https://cornac.readthedocs.io/en/latest/datasets.html#module-cornac.datasets.movielens">MovieLens 1M</a><br><a href="https://grouplens.org/datasets/movielens/"> source </a></td>
+    <td align="right">6,040</td>
+    <td align="right">3,706</td>
+    <td align="right">1,000,209</td>
+    <td align="center">INT<br>[1,5]</td>
+    <td align="center">&#10004;</td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><a href="https://cornac.readthedocs.io/en/latest/datasets.html#module-cornac.datasets.netflix">Netflix Small</a><br><a href="https://www.kaggle.com/netflix-inc/netflix-prize-data/"> source </a></td>
+    <td align="right">10,000</td>
+    <td align="right">5,000</td>
+    <td align="right">607,803</td>
+    <td align="center">INT<br>[1,5]</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><a href="https://cornac.readthedocs.io/en/latest/datasets.html#module-cornac.datasets.netflix">Neflix Original</a><br><a href="https://www.kaggle.com/netflix-inc/netflix-prize-data/"> source </a></td>
+    <td align="right">480,189</td>
+    <td align="right">17,770</td>
+    <td align="right">100,480,507</td>
+    <td align="center">INT<br>[1,5]</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><a href="https://cornac.readthedocs.io/en/latest/datasets.html#module-cornac.datasets.tradesy">Tradesy</a><br><a href="http://jmcauley.ucsd.edu/data/tradesy/"> source </a></td>
+    <td align="right">19,243</td>
+    <td align="right">165,906</td>
+    <td align="right">394,421</td>
+    <td align="center">BIN<br>{0,1}</td>
+    <td></td>
+    <td></td>
+    <td align="center">&#10004;</td>
+    <td></td>
+  </tr>
+</table>
+
+## Usage example
+
+Assume that we are interested in the FilmTrust dataset, which comes with both user-item ratings and user-user trust information. We can load these two pieces of information as follows,
```

Comment:
Assuming that we are interested in the FilmTrust dataset, which comes with both `user-item ratings` and `user-user trust` information. We can load these two parts of the data as follows

Reply:
Noted, I did minor changes, but mainly kept the original text.